rpt_base/Inventory.jrxml

```diff
@@ -0,0 +1,404 @@
+					<leftPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+					<bottomPen lineWidth="0.0" lineColor="#000000"/>
+					<rightPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+				</box>
+				<textElement textAlignment="Center" verticalAlignment="Middle"/>
+				<textFieldExpression><![CDATA[$F{MDSR_CODE}]]></textFieldExpression>
+			</textField>
+			<textField isBlankWhenNull="false">
+				<reportElement key="textField-8" x="40" y="-1" width="120" height="22" uuid="de714e10-0741-4045-84e1-12e17e6981bd"/>
+				<box>
+					<topPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+					<leftPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+					<bottomPen lineWidth="0.0" lineColor="#000000"/>
+					<rightPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+				</box>
+				<textElement textAlignment="Center" verticalAlignment="Middle"/>
+				<textFieldExpression><![CDATA[$F{MDSR_DESC}]]></textFieldExpression>
+			</textField>
+			<textField pattern="#,##0.00" isBlankWhenNull="false">
+				<reportElement key="textField-9" x="160" y="-1" width="130" height="24" uuid="ba83aa2d-a3f0-4b19-a0c8-da949ae318be"/>
+				<box>
+					<topPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+					<leftPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+					<bottomPen lineWidth="0.0" lineColor="#000000"/>
+					<rightPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+				</box>
+				<textElement textAlignment="Center" verticalAlignment="Middle"/>
+				<textFieldExpression><![CDATA[$F{LT_ID_A}==null?"":$F{LT_ID_A}]]></textFieldExpression>
+			</textField>
+			<textField pattern="#0" isBlankWhenNull="false">
+				<reportElement key="textField-10" x="370" y="-1" width="89" height="23" uuid="3cbbd069-d372-4aae-9462-2217e1904c53"/>
+				<box>
+					<topPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+					<leftPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+					<bottomPen lineWidth="0.0" lineColor="#000000"/>
+					<rightPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+				</box>
+				<textElement textAlignment="Center" verticalAlignment="Middle"/>
+				<textFieldExpression><![CDATA[$F{MINVTR_THEORETIC_QTY}]]></textFieldExpression>
+			</textField>
+			<textField pattern="#0" isBlankWhenNull="false">
+				<reportElement key="textField-13" x="460" y="-1" width="74" height="22" uuid="7055826f-b2ab-45ff-8a5c-19f4aa9b50c1"/>
+				<box>
+					<topPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+					<leftPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+					<bottomPen lineWidth="0.0" lineColor="#000000"/>
+					<rightPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+				</box>
+				<textElement textAlignment="Center" verticalAlignment="Middle"/>
+				<textFieldExpression><![CDATA[($P{printQtyReal}.intValue() == 1 ? "" : new DecimalFormat("0").format($F{MINVTR_REAL_QTY}) + "")]]></textFieldExpression>
```

Comment:
```suggestion
				<textFieldExpression><![CDATA[($P{printRealQty}.intValue() == 1 ? "" : new DecimalFormat("0").format($F{MINVTR_REAL_QTY}) + "")]]></textFieldExpression>
```

Reply:
Also, in order to simplify, it should be possible to specify the format on the field itself rather than in the expression